revision-history.yaml

```diff
@@ -5,7 +5,10 @@ revisionHistory:
   # additions to the specification should append entries here.
   thisVersion:
     spec:
+      - Add public modules.  Change circuits to support any number of public
+        modules which may have disjoint instance hierarchies.
     abi:
+      - Add ABI for public modules and filelist output.
```

Comment:
`- Changed ref ABI` (modulo phrasing?)

Reply:
(mostly "mention if something previously documented to work in a certain way has changed", however)

Reply:
Good catch. I added language indicating that the ABI for refs and groups changed in https://github.com/chipsalliance/firrtl-spec/pull/130/commits/924afa7e2b0add00e91cf90df5eaebfbe1280e8a.